src/test/java/com/google/api/codegen/testdata/discoveries/go/go_logging.v2beta1.json.baseline

```diff
@@ -22,25 +25,33 @@ import (
 
 func main() {
   ctx := context.Background()
+
+  // Authentication is provided by the `gcloud` tool when running locally
```

Comment:
Hmmm ok, I'll make note

> Is this comment the same across languages?
Yes

@vchudnov-g Do you have a strong opinion? I think it's reasonable to remove the `gcloud` tool comment, as it isn't particularly informative without looking at the link